test/distributed/test_c10d_functional_native.py

```diff
@@ -84,7 +88,7 @@ def _init_process_group(self) -> None:
         torch._C._distributed_c10d._register_process_group("default", dist.group.WORLD)
 
     @skip_if_lt_x_gpu(2)
-    def test_all_reduce(self) -> None:
+    def test_all_reduce_single(self) -> None:
```

Comment:
Curious what is `all_reduce_single` mean?

Reply:
It just means that it's not coalesced. The purpose is to make pattern matching the test easier with `-k`.